util/db/repository_secrets_test.go

```diff
@@ -931,6 +931,12 @@ func TestSecretsRepositoryBackend_GetAllHelmRepoCreds(t *testing.T) {
 }
 
 func TestRepoCredsToSecret(t *testing.T) {
```

Comment:
Missing unit test to validate the behavior for a write secret and test getRepoCredSecretType